internal/tpm/session.go

```diff
@@ -65,8 +66,14 @@ func (s *tpmSession) initialize() error {
 	if err != nil {
 		return fmt.Errorf("ensuring SRK: %w", err)
 	}
-	s.srk = srkHandle
 	s.handles[SRK] = srkHandle
+	// flush the SRK after initialization to free up transient space. The SRK
+	// isn't needed after loading.
```

Comment:
you mean loading the handle into the session struct, not loading into the TPM, right?

Reply:
`isn't needed after 'loadExistingKeys'` is probably a better comment. Once the LAK and LDevID are loaded into the TPM, we have no use for the SRK (as of now)